source/includes/job-board/_applications.md

```diff
@@ -217,3 +238,26 @@ longitude for the selected location using the [Google Place Details API](https:/
 this API call.
 
 Note that all 3 fields must be included. If only `location` is sent and `latitude` and `longitude` are omitted, `location` will be ignored entirely.
+
+### Validations on POST Requests
+
+**Job validations:**
+
+* The job must be live and published
+* The job board must be live and published
+* The job post ID must exist on the job board with the supplied board_token
+
+**Field validations:** 
+
+* `first_name`, `last_name`, and `phone` fields must not contain a URL
```

Comment:
Should we make a comment about how:

first name, last name and email are required?